package.json

```diff
@@ -5,6 +5,8 @@
   "dependencies": {
     "bulma": "^0.8.0",
     "jsonp": "^0.2.1",
+    "react-icons": "^3.9.0",
+    "react-scroll-up": "^1.3.3",
```

Comment:
I don't know if we need these libraries. For other icons we used the svg directly (you can find free svg icons here https://icomoon.io/app/#/select )

Reply:
Also for scrolling to top did you try window.scrollTo function? I think it would be better for the bundle size if this solution works